services/identity/struct.go

```diff
@@ -221,7 +221,7 @@ type ProposeVote struct {
 
 // PropagateIdentity sends a new identity to other identityServices
 type PropagateIdentity struct {
-	*storage
+	*Storage
```

Comment:
Because it will be sent over the network, so it needs to be `BinaryMarshal`lable.